src/components/displayError.js

```diff
@@ -0,0 +1,15 @@
+import { createHeader } from '../layouts/createLayout.js';
+import { createFooter } from '../layouts/createLayout.js';
+import { clearBody } from '../components/clearBody.js';
+
+export function displayError(error) {
```

Comment:
@Mmarvinn It's not needed you can use your `renderPageLayout` function. Just like this 
```js
renderPageLayout(`Error status: ${error.status}`)
```

Reply:
fix